packages/webdriverio/tests/commands/browser/$$.test.js

```diff
@@ -0,0 +1,45 @@
+import request from 'request'
+import { remote } from '../../../src'
+
+describe('elements', () => {
+    let browser
+    let elems
+
+    beforeAll(async () => {
+        browser = await remote({
+            baseUrl: 'http://foobar.com',
+            capabilities: {
+                browserName: 'foobar'
+            }
+        })
+    })
+
+    it('should have a sessionId when instance was created', () => {
+        expect(browser.sessionId).toBe('foobar-123')
+        expect(request.mock.calls).toHaveLength(1)
+        expect(request.mock.calls[0][0].method).toBe('POST')
+        expect(request.mock.calls[0][0].uri.path).toBe('/wd/hub/session')
+    })
```

Comment:
We have this test in all files, I don't think this is needed

Reply:
Good catch.  Late evening copy paste :) . Looks like this already exists in ```module.test.js``` so I'm going to kill it from all other test files